x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_add.tsx

```diff
@@ -134,7 +134,7 @@ export const AlertAdd = ({
       const newAlert = await createAlert({ http, alert });
       toastNotifications.addSuccess(
         i18n.translate('xpack.triggersActionsUI.sections.alertAdd.saveSuccessNotificationText', {
-          defaultMessage: "Saved '{alertName}'",
+          defaultMessage: 'Alert was created',
```

Comment:
> Also wondering if we should add the name back to the message - at some point someone thought there was value in doing that.

Not opposed to this:

```suggestion
          defaultMessage: 'Alert was created ({alertName})',
```

Reply:
'Create' is the correct wording here. The preferred format for toasts is:

Alert ({alertName}) was created

In the example above, the toast would be:

Alert 'Transaction duration threshold | opsbean-node' was created.

However, if alert names are typically that long, this form is better:

Created alert 'Transaction duration threshold | opsbean-node'

Reply:
At least for APM the names are quite long, which is why I prefer your second suggestion:
> Created alert 'Transaction duration threshold | opsbean-node'

Reply:
```suggestion
          defaultMessage: 'Created alert ({alertName})',
```